samples/nodejs/nextjs-boilerplate/README.md

```diff
@@ -0,0 +1,13 @@
+This template is a simple static web page project developed using Next.js. It is ready to be deployed by Defang, and you could make your project available to the world in just 5 minutes!
+
+# Prerequisite
+1. Download <a href="https://github.com/defang-io/defang">Defang CLI</a>
+2. Create a <a href="https://docs.docker.com/develop/develop-images/dockerfile_best-practices/">Dockerfile</a>
+3. Create a <a href="https://docs.docker.com/compose/gettingstarted/">docker-compose file</a>
```

Comment:
I'm not sure I understand this. The sample already includes a `Dockerfile` and a `docker-compose` file. 

It might make sense to add something like "Optional: Install Docker to run the sample locally"

Reply:
I thought the README is more like a general guide for users to deploy something with Defang, but since users need a dockerfile and a docker-compose file to properly deploy it. I thought I should mention it in the guide. I understand that it might be confusing since the READMEs are not very case specific, as this sample already has a dockerfile and a docker-compose. Should I explain it a bit to clarify (e.g a dockerfile and a docker-compose is needed if you deploy it via Defang; in this sample, we have built it for you.)